FogCreek/Glitch-Community
# Fallacious slice: Rename the DEPLOY_ENV var to BUILD_TYPE

## Links
https://fallacious-slice.glitch.me/

## Changes:
- `DEPLOY_ENV=production|development|ci` is now `BUILD_TYPE=watcher|memory|static`
- Added a section to the readme listing env vars and what they do

## How To Test:
Try playing with the site and make sure it still builds correctly in all configs

## Feedback I'm looking for:
Are these names intuitive?

## Things left to do before deploying:
- Update the env vars on ~community and ~community-staging when I deploy this
- I probably need to change something for ci, where is that?
